pkgs/tools/networking/octodns/providers/cloudflare/default.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  octodns,
+  pytestCheckHook,
+  pythonOlder,
+  requests,
+  requests-mock,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "octodns-cloudflare";
+  version = "0.0.9";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```

According to their [`setup.py`](https://github.com/octodns/octodns-cloudflare/blob/v0.0.9/setup.py#L51) this package requries at least Python 3.9

---

I squashed the commits and both commit messages are listed in the single squashed commit, so actually there is both the maintainers commit and the package commit squashed into one commit. Also, I now have updated the python version.